app/models/user.rb

```diff
@@ -393,6 +393,19 @@ def onboarding
     @onboarding ||= VeteranOnboarding.for_user(self)
   end
 
+  def demographics_info
+    # return nil unless DemographicsPolicy.new(self).access? && MPIPolicy.new(self).queryable?
+
+    @demographics_info ||= VAProfileRedis::Demographics.for_user(self)
+  rescue => e
+    Rails.logger.info('[User] VAProfileRedis::Demographics error', e.message)
```

Comment:
Other loggers include an ICN as the last argument. Unsure if we should include ICN, maybe also make this a warning? I basically looked at what was going on in `log_mhv_user_account_error`… other code in the file doesn't log service errors from within the User model.